### ISSUE TITLE
Make key "tap area" size configurable for people with fat fingers or hand tremors.

### ISSUE THREAD

**Issue by brycebba** [futo-org/android-keyboard#115]
I like that the keyboard has some size control but I have a hard time with tapping the wrong letters because the tap footprint area for letters are very close. This is a trivial problem and probably not a problem for most people. I have hand tremors and it actually helps a lot when the tap area is reduced so its closer to the center of the keys with a little "dead zone" near the perimeter.

**Comment by abb128** [comment]
If I understand correctly you want taps to not register if they're too far between keys? Are there any other keyboards with the feature you're describing?

**Comment by brycebba** [comment]
> If I understand correctly you want taps to not register if they're too far between keys? Are there any other keyboards with the feature you're describing?

Yes you are correct. I had made some keyboards using Anysoft Keyboard a couple years ago that accomplished this by putting qwerty keys in traditional layout but reduced the key size which increased the padding between them. I didn't decrease the size of the letter on the key, just decreased the area that could trigger that key tap event. Hopefully that makes sense
